### PR TITLE
Some fixes for the ffg data

### PR DESCRIPTION
**Pull request by therealppa**
I left out the R2 Astromech cost on purpose, thought it would be risky to fiddle with costs.

### CODE REVIEW

File: scripts/ffgprocess.js [hunk 1]
```diff
@@ -398,17 +398,34 @@ function processCard(card) {
...
-    }
-    // Lando's Millennium Falcon [Title] doesn't properly capitalize ship name
-    if (card.id === 390) {
-      card.ability_text = card.ability_text.replace(
-        "escape craft",
-        "Escape Craft"
-      );
+    switch (card.id) {
+      case 282:
+        // card.cost.values['1'] = 4;
```
Comment (guidokessels) [inline scripts/ffgprocess.js:403]: This is commented out. Can it be removed?

File: scripts/ffgprocess.js [hunk 2]
```diff
@@ -466,10 +483,7 @@ function processCard(card) {
   } else {
     // Card-specific tweaks:
     //
-    if (card.id === 597) {
-      // Odd Ball [BTL-B Y-wing]: Card name is "Oddball" which should be "Odd Ball"
-      card.name = '"Odd Ball"';
```
Comment (guidokessels) [inline scripts/ffgprocess.js:471]: I see FFG has fixed Odd Ball's name in the new API. Nice!

**Review by guidokessels — CHANGES_REQUESTED** [review]
Thanks for adding the changes from Errata. Can you check what needs to be done with the line that's commented out?

File: scripts/ffgprocess.js (hunk 1 above)
Comment (therealppa) [reply scripts/ffgprocess.js]: removed it

**Comment by guidokessels** [comment]
Thanks!

**Pull request merged**